lib/pushmi_pullyu/cli.rb

```diff
@@ -225,7 +229,7 @@ def shutdown
       exit!(1)
     else
       # using stderr instead of logger as it uses an underlying mutex which is not allowed inside trap contexts.
-      $stderr.puts 'Exiting...  Interrupt again to force quit.'
+      warn 'Exiting...  Interrupt again to force quit.'
```

Comment:
theres a test that makes this fail. Requiring updating it. Look at travisCI build

Reply:
Take a peak at changing the test to using these maybe?

https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/output-matcher